.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
If you want to prevent empty issues, you'd set `blank_issues_enabled` to `false`, but that's something to do later. 😉

Reply:
```suggestion
blank_issues_enabled: false
```